sci-mathematics/gimps/gimps-30.4.8.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit pax-utils readme.gentoo-r1 systemd tmpfiles
+
+DESCRIPTION="The Great Internet Mersenne Prime Search"
+HOMEPAGE="https://www.mersenne.org/"
+
+# The author has finally caved in to the pressure of putting the build
+# number in the package name. Some dark magic necessary to get to the
+# package name
+
+MY_PV=$(ver_rs 2 'b' )
+
+SRC_URI="
+	amd64? ( https://www.mersenne.org/ftp_root/gimps/p95v${MY_PV/./}.linux64.tar.gz )
+	"
+
+SLOT="0"
+LICENSE="GIMPS"
+RESTRICT="mirror bindist"
+KEYWORDS="-* ~amd64"
+
+# Since there are no statically linked binaries for this version of mprime,
+# and no static binaries for amd64 in general, we use the dynamically linked
+# ones and try to cover the .so deps with the packages listed in RDEPEND.
+# libgmp.so.10.3.2 is bundled within the .tar.gz, but we use the system one.
+
+DEPEND=""
+RDEPEND="net-misc/curl
+	>=dev-libs/gmp-6.1.2"
+
+S="${WORKDIR}"
+I="/opt/gimps"
```

Comment:
This is too disambiguous and should reflect the meaning of variable better. Like `OPTINSTALLDIR`.